index.js

```diff
@@ -72,7 +72,8 @@ module.exports = options => {
 		session.protocol.registerFileProtocol(options.scheme, handler);
 	});
 
-	return async window_ => {
-		await window_.loadURL(`${options.scheme}://${options.hostname}`);
+	return async (window_, query) => {
+		const queryString = query ? `?${query}` : '';
```

Comment:
This is not safe. Would be better to accept a `searchParams` object that is passed `new URLSearchParams`.

This is also missing docs, types, and test.

Reply:
Sorry I'm not completely following. Do you mean something like this:

```js
return async (window_, searchParams) => {
  const qs = searchParams ? '?' + new URLSearchParams(searchParams).toString() : ''
  await window_.loadURL(`${options.scheme}://${options.hostname}${qs}`);
};
```

Reply:
Yes 👍